googlespanner/README.md

```diff
@@ -0,0 +1,63 @@
+<!--
+Copyright (c) 2017 YCSB contributors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you
+may not use this file except in compliance with the License. You
+may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+implied. See the License for the specific language governing
+permissions and limitations under the License. See accompanying
+LICENSE file.
+-->
+
+# Google Spanner Driver for YCSB
+
+## Limitations and caveats
+
+* Only Application Default Credentials are supported. In other words,
+  if running on a Google Cloud instance with permissions granted when
+  creating it, authentication will seamlessly work. But use of
+  explicit keys is not implemented.
```

Comment:
Adjusted to hopefully be clearer, and link to documentation.

Yes, like an AWS access key.